plugins/server/dev.hayden/khealth/3.0/documentation.md

```diff
@@ -0,0 +1,115 @@
+# KHealth
+[![Release](https://jitpack.io/v/haydenmeloche/khealth.svg)](https://jitpack.io/#dev.hayden/khealth)
```

Comment:
```suggestion
```

The header will automatically appear in the generator, and badges will not work in our markdown.